Makefile

```diff
@@ -1,5 +1,13 @@
-.DEFAULT_GOAL := test
+lint:
+	# Coding style static check.
+	@go install honnef.co/go/tools/cmd/staticcheck@latest
+	@go mod tidy
+	#staticcheck `go list ./...`
```

Comment:
Do you want to put the staticcheck in its own target? That would make more sense, IMO.

Reply:
I prefer having one target that gathers all vetting scripts. Except if there is a case where we need to launch staticcheck but not the other linting tools.